agimus_demo_07_deburring/README.md

```diff
@@ -0,0 +1,27 @@
+AGIMUS demo 05 pick and place
```

Comment:
Copy pasted from demo 5 I guess ;)

Reply:
Maybe it was copied, maybe it was not... We will never know... Anyway I briefly updated the demo